src/Altinn.AccessManagement/Controllers/BFF/DelegationsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Altinn.AccessManagement.Core.Models;
+using Altinn.AccessManagement.Core.Models.ResourceRegistry;
+using Altinn.AccessManagement.Core.Services.Interfaces;
+using Altinn.AccessManagement.Filters;
+using Altinn.AccessManagement.Models;
+using Altinn.AccessManagement.Models.Bff;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.AccessManagement.Controllers.BFF
+{
+    /// <summary>
+    /// Controller responsible for all operations for managing delegations of Altinn Apps
+    /// </summary>
+    [ApiController]
+    [AutoValidateAntiforgeryTokenIfAuthCookie]
+    [ApiExplorerSettings(IgnoreApi = true)] 
+    public class DelegationsController : ControllerBase
+    {
+        private readonly ILogger _logger;
+        private readonly IPolicyInformationPoint _pip;
+        private readonly IPolicyAdministrationPoint _pap;
+        private readonly IDelegationsService _delegation;
+        private readonly IMapper _mapper;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DelegationsController"/> class.
+        /// </summary>
+        /// <param name="logger">the logger.</param>
+        /// <param name="policyInformationPoint">The policy information point</param>
+        /// <param name="policyAdministrationPoint">The policy administration point</param>
+        /// <param name="delegationsService">Handler for the delegation service</param>
+        /// <param name="mapper">mapper handler</param>
+        public DelegationsController(
+            ILogger<DelegationsController> logger,
+            IPolicyInformationPoint policyInformationPoint,
+            IPolicyAdministrationPoint policyAdministrationPoint,
+            IDelegationsService delegationsService,
+            IMapper mapper)
+        {
+            _logger = logger;
+            _pap = policyAdministrationPoint;
+            _pip = policyInformationPoint;
+            _delegation = delegationsService;
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Endpoint for retrieving delegated resources between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpGet]
+        [Authorize]
+        [Route("accessmanagement/api/v1/bff/{who}/delegations/maskinportenschema/inbound")]
```

Comment:
Maybe change api to bffapi so the url change is easier, and it is easier to differentiate from the real api

Reply:
ya, updated to bffapi